deploys/demo.syncfor.science/config/vendors/Allscripts.yml

```diff
@@ -1,43 +1,49 @@
 ---
 api:
-  url: https://apis-test.followmyhealth.com/fhir-uat/api/
-  patient: 3dcfa065-5a1e-4577-aa04-a719013f736b
+  versions:
+    DSTU2:
+      url: https://apis-test.followmyhealth.com/fhir-uat/api/
+      patient: 3dcfa065-5a1e-4577-aa04-a719013f736b
 
-auth:
-  strategy: authorization_code
-  client_id: 90d4ed27-4790-4a3e-80c2-436d1942d680
-  # client_secret intentionally included
-  client_secret: Ym0ZYTtSjrQM7ZqLfkMKuj1exRmuIKbBCQR+HDoDQgtEb4ZiDhyE1dDk9S3h2vfMRdxvzV18KMhv7Fz0dPiP6w==
-  scope: launch/patient patient/*.read offline_access
-  confidential_client: True
-  sign_in_steps:
-    - element: "[translate='DEFAULT_LOGIN_BUTTON']"
-      click: True
-      optional: True
-    - element: "#UserName"
-      send_keys: "floppyrobot10"
-    - element: "#Password"
-      send_keys: "s4s_pilot!"
-    - element: "[translate='Login_LogIn']"
-      click: True
-  authorize_steps:
-    - element: "#permission-component"
-      wait: 10
-      optional: True
-      execute_script: |
-        Array.from(document
-          .querySelectorAll('span > label > input[type="checkbox"]'))
-          .forEach(function(el) { el.click(); });
-    - element: "span:first-child button"
-      optional: True
-      click: True
-    - element: "span:first-child button"
-      optional: True
-      click: True
-  revoke_url: https://mutestcertification.azurewebsites.net/api/Removal
-  revoke_steps:
-    - element: 'button.removal-button[data-client-id="90d4ed27-4790-4a3e-80c2-436d1942d680"]'
-      click: True
-      wait: 10
+use_cases:
+  security: DSTU2
+  ehr: DSTU2
 
-version: DSTU2
\ No newline at end of file
+auth:
+  versions:
```

Comment:
In this case: do we ever expect to an auth recipe for >1 version in the same config file? I don't think so, but I may be missing something. (Again, it's okay to have an extra node in the tree here for consistency with other branches -- just want to make sure I'm following.)

Reply:
I didn't expect it, no. I was trying to be explicit as to which version everything was using.